### PR TITLE
Create CODESOWNER

### PR DESCRIPTION
**Pull request by lisna02**
### Changes

Closes:

### Task list

- [ ] For workflow changes, I have verified the Actions workflows function as expected.
- [ ] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by AdrienneAEW — COMMENTED** [review]
Looks good...Not really sure what to say here.

**Review by AdrienneAEW — APPROVED** [review]
I say we go with the changes. The game over message is much more descript and user friendly.

**Review by AdrienneAEW — COMMENTED** [review]
I'm commenting again...Not working though.

File: CODESOWNER [hunk 1]
```diff
@@ -0,0 +1 @@
+Addinga new file
```
Comment (AdrienneAEW) [inline CODESOWNER:1]: New file is good to go...

**Review by AdrienneAEW — COMMENTED** [review]
This works for me.

File: CODESOWNER (hunk 1 above)
Comment (izeman87) [reply CODESOWNER:1]: CODESOWNER

Comment (izeman87) [reply CODESOWNER:1]:
```suggestion
Addinga new file
```

**Review by izeman87 — APPROVED** [review]

**Review by izeman87 — COMMENTED** [review]
LJ

**Review by izeman87 — APPROVED** [review]

**Review by izeman87 — APPROVED** [review]

**Review by izeman87 — APPROVED** [review]

**Review by izeman87 — APPROVED** [review]

**Review by Bossman74222 — APPROVED** [review]

**Review by Wangtt185 — APPROVED** [review]

**Review by Wangtt185 — COMMENTED** [review]
1

**Review by Wangtt185 — COMMENTED** [review]
1

**Pull request closed without merging**